themes/acquia_claro/.gitignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
Hey @vishalkhode1 

I see node_modules/ added in .gitignore in acquia_cms root .gitignore file but not in themes/acquia_claro. Hence, I thought it is important to add .dedicated gitignore file in theme as well since we are separating out the theme in a different repo.

Please suggest if this is correct or it needs to be changed/removed.

Reply:
Yes, looks correct.